energymap/api_options.go

```diff
@@ -0,0 +1,25 @@
+package energymap
+
+import "net/http"
+
+func WithHTTPClient(h *http.Client) ApiOption {
+	return func(a *ApiClient) error {
+		a.client = h
+		return nil
+	}
+}
+
+func WithThresholds(moderate, high float64) ApiOption {
```

Comment:
This part isn't clear to me. What are these thresholds used for? Also what about adding a comment explaining their purpose?

Reply:
Short answer: dangling code, I'll remove it.

Long answer for all of these: on main thread